<launch>

    <arg name="name" default="testdemo"/>

    <arg name="config_file" value="$(find lfd_interface)/config/demonstrations/default.yaml"/>

    <node name="keyboard_command" pkg="lfd_interface" type="keyboard_command.py">
    </node>

    <include file="$(find lfd_interface)/launch/lfd_storage.launch"/>
    
    <node name="lfd_recorder_node" pkg="lfd_interface" type="lfd_recorder_node" output="screen">
        <param name="demonstration_name" value="$(arg name)"/>
        <rosparam command="load" file="$(arg config_file)" />
    </node>

</launch>
